PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -22,6 +27,10 @@ if (match) {
 		file_system = new ActiveXObject('Scripting.FileSystemObject'),
 		file = decodeURIComponent(match[1]).replace(/\+/g, ' '),
 		search_path = file.replace(/\//g, '\\');
+		
+	if (settings.project_basepath != '' && settings.project_pathalias != '') {
+		file = file.replace(settings.project_basepath, settings.project_pathalias);

```

Comment:
If I replace it with an empty string in Windows, this results in the following exception:

    Cannot find file 'C:\Program Files (x86)\Google\Chrome\Application\45.0.2454.93\var\www\project\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\DebugHandlersListener.php'